### ISSUE TITLE
Build regression for the 11.30.5 release

### ISSUE THREAD

**Issue by bradchoate** [auth0/lock#2036]
### Describe the problem

It appears the 11.30.5 release is missing the "build" directory. I suspect this has been discovered already, but just wanted to formally report it. 11.30.4 is fine.

### What was the expected behavior?

The build tree should exist with minified, localized assets.

### Reproduction

```
$ yarn add auth0/lock#11.30.4
$ ls -l node_modules/auth0-lock/build
ls: node_modules/auth0-lock/: No such file or directory
```

### Environment

- Version 11.30.5

**Comment by stevehobbsdev** [comment]
Thanks for the report @bradchoate, a fix is in progress. Will report back once it's out but please pin to 11.30.4 for now until a resolution is in place 👍🏻

**Comment by stevehobbsdev** [comment]
Sorry @bradchoate, I had gotten confused with a separate issue. It's expected that the build folder is not included here. If follow your steps and do:

```
yarn add auth0-lock@11.30.4
ls node_modules/auth0-lock
```

I don't get a build folder there, which is expected since it's ignored in the `.npmignore` folder. Rather, bundlers should be pointed to `lib/index.js` by [our package.json file](https://github.com/auth0/lock/blob/eccda0000dc1644e575e017b00d30ba4c05f025a/package.json#L20).

I'm interested to know what caused you to raise this though, were you getting an error on build after you'd installed the updated package?

**Comment by bradchoate** [comment]
Sorry, I didn't provide the right reproduction steps. I did notice the release .zip files differ in this regard. A "build" directory in 11.30.4, but none in 11.30.5, so I assumed a `yarn add` command would include one for 11.30.4. Here's our actual dependency:

    "@bower_components/auth0-lock": "auth0/lock#^11.30.2"

I think that kind of reference causes it to pull the package via GitHub rather than npm. The `@bower_components` portion is just to have the package installed within a particular tree of node_modules, which our Gruntfile handles differently for front-end versus back-end modules.

You can observe the different behavior by doing:

    $ yarn add auth0/lock#11.30.4

and see that a "build" directory is included in `node_modules/auth0-lock`, but:

    $ yarn add auth0/lock#11.30.5

will not contain a "build" directory.

**Comment by stevehobbsdev** [comment]
Thanks. The fact you're using Bower makes a difference and yes, I'm aware that the source code asset attached to the release no longer includes the `build/` folder. Again, thanks for reporting.

Can I ask about your use case for using Bower - what is the blocker to using the NPM package or even the CDN to include this library? I'm asking because one of the things we're looking at is whether to continue supporting Bower or not given that [it's deprecated](https://snyk.io/blog/bower-is-dead/), and more or less obsolete at this point considering all the other tools available, but we don't want to do that of course if we know that lots of people rely on it.

**Comment by bradchoate** [comment]
We aren't actually using Bower for packages now. We were, but using yarn now and just using this convention to clearly delineate the modules that are front-end versus back-end.

**Comment by stevehobbsdev** [comment]
@bradchoate This should now be fixed in the 11.30.6 release, if you'd like to give it a try and report back if you're still having issues.

**Comment by bradchoate** [comment]
Looks great! Thank you.
